halo/app/app_config.go

```diff
@@ -53,6 +59,14 @@ const (
 
 //nolint:gochecknoglobals // Cosmos-style
 var (
+	deliveryInterval = func(network netconf.ID) int64 {
+		if network.IsProtected() {
+			return 20_000 // roughly ~12h assumping 0.5bps finalization rate
+		}
+
+		return 2
+	}
```

Comment:
Move this to a normal func declaration it doesn't have to be a variable, remove the magic numbers. 

```
var (
    protectedDeliveryInterval = 20_000 // roughly ~12h assumping 0.5bps
    ephermeralDeliveryInterval = 2
    ....
)

func deliveryInterval(network netconf.ID) int64 {
	if network.IsProtected() {
		return protectedDeliveryInterval
	}

	return ephermeralDeliveryInterval
}
```

Reply:
agreed. But rather use the same prefix for similar consts/vars to group them together.
```
deliveryIntervalProtected = 20_000 // roughly ~12h assumping 0.5bps 
deliveryIntervalEphemeral = 2 // Fast updates while testing
```

Reply:
Also maybe add a note the const block in `app_config`:
```
// Note that these constants form part of consensus logic and can only be changed during network upgrades.
const (
   ...
)
```